blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseContainerIntegrationTest.java

```diff
@@ -553,7 +553,7 @@ protected void addTenObjectsUnderPrefix(String containerName, String prefix) thr
 
    protected void awaitConsistency() {
       if (view.getConsistencyModel() == ConsistencyModel.EVENTUAL) {
-         Uninterruptibles.sleepUninterruptibly(10, TimeUnit.SECONDS);
+         Uninterruptibles.sleepUninterruptibly(20, TimeUnit.SECONDS);
```

Comment:
I like the idea of using eventuallyTrue for testing, but I thought it was deprecated/removed?
Seems to be the most efficient way to do it, though.